src/Sound/Tidal/Boot.hs

```diff
@@ -107,6 +141,14 @@ only = (hush >>)
 p :: Tidally => ID -> ControlPattern -> IO ()
 p = streamReplace tidal
 
+-- | Silences a specific stream, regardless of ControlPattern input. Useful for rapid muting of streams
+_p :: Tidally => ID -> ControlPattern -> IO ()
+_p n _ = streamReplace tidal n silence
```

Comment:
Might be good to use a parameter other than `n` here to avoid the warning? The Stream module functions use `k` (for pattern key) a lot in this situation

Reply:
Sure! Just changed it